monasca/values.yaml

```diff
@@ -367,6 +367,186 @@ alarms:
     os_password: password
     os_project_name: mini-mon
     os_project_domain_name: Default
+  definitions_configuration:
+     definitions.yml.j2: |
+      notifications:
+        - name: "{{ NOTIFICATION_NAME | default('default') }}"
+          type: "{{ NOTIFICATION_TYPE | default('email') }}"
+          address: "{{ NOTIFICATION_ADDRESS | default('root@localhost') }}"
+
+      alarm_definitions:
+        - name: "Host Status"
```

Comment:
we should really change the default alarm definitions for metrics that are there